android/app/src/main/java/com/newspringchurchapp/MainActivity.java

```diff
@@ -1,24 +1,12 @@
 package com.newspringchurchapp;
-
-import android.content.Intent;
-import android.net.Uri;
 import android.os.Bundle;
-import org.devio.rn.splashscreen.SplashScreen;
 import com.facebook.react.ReactActivity;
 import com.facebook.react.ReactActivityDelegate;
 import com.facebook.react.ReactRootView;
 import com.swmansion.gesturehandler.react.RNGestureHandlerEnabledRootView;
+import com.zoontek.rnbootsplash.RNBootSplash;
 
 public class MainActivity extends ReactActivity {
-  @Override
-  protected void onCreate(Bundle savedInstanceState) {
-      SplashScreen.show(this, R.style.SplashScreenTheme);
-      super.onCreate(savedInstanceState);
-      // ATTENTION: This was auto-generated to handle app links.
-      Intent appLinkIntent = getIntent();
-      String appLinkAction = appLinkIntent.getAction();
-      Uri appLinkData = appLinkIntent.getData();
```

Comment:
did you test that content links still open the app? seems like this is a code that handles that.

Reply:
If by "content links" you mean links like on the Connect tab then yes. They still work.

Reply:
I mean do newspring.cc links still open in-app on android?

Reply:
Yep. I only tested the ones on the Connect tab and they seem to work just fine.

Reply:
no from outside coming in. like from a text message.

Reply:
Oh, yeah, no. I didn't test that on either platform. Can we even do that in a sim? I don't remember.

Reply:
I can definitely just put that part back to make sure we're good.